scripts/docker_cmake

```diff
@@ -32,6 +32,7 @@ ARGS=(
     ### PACKAGES CONFIGURATION ###
     -D Trilinos_ENABLE_ALL_PACKAGES=OFF
     -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES=OFF
+    "${Trilinos_ENABLE_Fortran}"
```

Comment:
Do `-D Trilinos_ENABLE_Fortran=${ENABLE_FORTRAN}`, and then just use `ENABLE_FORTRAN="ON"` or "OFF" in your `docker_clang_env.sh`.